### PR TITLE
Convert to pdk

### PR DESCRIPTION
**Pull request by luoymu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
```
Comment (edestecd) [inline CHANGELOG.md]: Can you convert our old CHANGELOG to CHANGELOG.md.

Example: https://github.com/edestecd/puppet-mariadb/commit/0fd43c12bb3ce0e0ff2d098ddf7e6560b0ef3416

File: .sync.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+mock_with: ':rspec'
+
+.gitlab-ci.yml:
+  delete: true
+appveyor.yml:
+  delete: true
+
+.travis.yml:
+  user: MiamiOH
+  secure: "RMGe+qbnUvk2tM7bXlLObXjlNB+C1cyHOU3FGJS0GdESVkdI6Wne66Y6tI5XPlWkI4UA0YWU/38fJllLuOdbKooNg5XOefocc5PQbmDbi8A7jRYMiZx7J9lblJKqb0HspmVsrWzz1Ss8l/UmkQWWAUAHU4yHTPquqFQV19CZytE="
```
Comment (edestecd) [inline .sync.yml]: We need too generate a secure key that is specific to this repo with the `travis encrypt` command

File: metadata.json [hunk 3]
```diff
@@ -7,38 +7,68 @@
...
   "project_page": "https://github.com/MiamiOH/puppet-pam_access",
   "issues_url": "https://github.com/MiamiOH/puppet-pam_access/issues",
-  "tags": ["pam", "access"],
-  "description": "The pam_access module allows access control based on username, groupname, network address, and process name.  This module allows Puppet to manage entries in the access.conf file.",
   "dependencies": [
-    {"name": "puppetlabs/stdlib", "version_requirement": ">= 4.1.0 < 7.0.0"},
-    {"name": "herculesteam/augeasproviders_pam", "version_requirement": ">= 2.1.0 < 3.0.0"}
+    {
+      "name": "puppetlabs/stdlib",
+      "version_requirement": ">= 4.1.0 < 6.0.0"
```
Comment (edestecd) [inline metadata.json:13]: this has changed from 7.0.0 to 6.0.0. Did we want this?

File: spec/classes/pam_access_spec.rb [hunk 4]
```diff
@@ -1,29 +1,28 @@
...
+      let(:params) { { manage_pam: true } }
 
-      it { should compile.with_all_deps }
+      it { is_expected.to compile.with_all_deps }
 
       it do
-        should contain_exec('authconfig-access').with(
-          :command => '/usr/sbin/authconfig --enablelocauthorize --enablepamaccess --update'
+        is_expected.to contain_exec('authconfig-access').with(
+          'command' => '/usr/sbin/authconfig --enablelocauthorize --enablepamaccess --update',
```
Comment (edestecd) [inline spec/classes/pam_access_spec.rb]: `command: '/usr/sbin/authconfig --enablelocauthorize --enablepamaccess --update',`

File: metadata.json (hunk 3 above)
Comment (luoymu) [reply metadata.json:13]: yes, One step a time

**Pull request merged**